Binary_Search_Trees/Binary_Search_Tree.kt

```diff
@@ -0,0 +1,106 @@
+class BinarySearchTree {
+	private var head: node? = null
+
+	class node(var data: Int) {
+		var left: node? = null
+		var right: node? = null
+
+	}
+
+	fun Insert(value: Int) {
+		val temp = node(value)
+		if (head == null)
+			head = temp
+		else {
+			var current: node?
+			current = head
+			while (current != null) {
+				if (value < current.data) {
+					if (current.left != null)
+						current = current.left
```

Comment:
please keep four spaces, don't use tab. Do this correction in your entire code

Reply:
Can you please check now? Thank you